api/screenshot.js

```diff
@@ -2,18 +2,40 @@ const chromium = require('@sparticuz/chromium-min');
 const puppeteer = require('puppeteer-core');
 let _page;
 
+// get platform specific chrome executable for local environment
+const exePath = process.platform === "win32"
+        ? "C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe"
+        : process.platform === "linux"
+        ? "/usr/bin/google-chrome"
+        : "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome";
+
+async function getOptions(isDev) {
+    let options;
+    console.log('exePath:', exePath);
+    if (isDev) {
+        options = {
+        args: [],
```

Comment:
Is there a reason why `args` is locally different than in prod?

Reply:
done